docs/design/adrs/2024-04-23-implement-prompflow-orchestration.md

```diff
@@ -0,0 +1,42 @@
+# Implement a PromptFlow Orchestration as an alternative for the chat backend
```

Comment:
The title of the document suggests you think we **do** want to implement this. 
```suggestion
# Do not implement a PromptFlow Orchestration as an alternative for the chat backend
```
Would need ot rename the file as well

Reply:
How about `2024-04-23-prompflow-orchestration.md`

Reply:
Not applicable anymore since the decision matches with the file name.